include/wx/charts/wxchartlabel.h

```diff
@@ -55,6 +55,8 @@ class wxChartLabel : public wxChartElement
     wxChartLabel(const wxString &text, wxDouble width,
         wxDouble height, const wxChartLabelOptions &options);
 
+    wxChartLabel &operator=(const wxChartLabel &obj);
```

Comment:
If we are adding an assignment operator we should also add a copy constructor.

Reply:
I've added the copy constructor to the header file